XCUITests/FxScreenGraph.swift

```diff
@@ -143,6 +143,12 @@ class Action {
     static let DisablePasscodeTypeIncorrectPasscode = "DisablePasscodeTypeIncorrectPasscode"
 
     static let TogglePocketInNewTab = "TogglePocketInNewTab"
+    static let ToggleBookmarksInNewTab = "ToggleBookmarksInNewTab"
+    static let ToggleHistoryInNewTab = "ToggleHistoryInNewTab"
+
+    static let ShowBlankPage = "ShowBlankPage"
+    static let ShowBookmarksPage = "ShowBookmarksPage"
+    static let ShowHistoryPage = "ShowHistoryPage"
```

Comment:
I think these should be named as `SelectNewTabAsBlankPage` or something to suggest that they're Settings actions.

Reply:
👍 Agree, will change the name